static/bidder-info/adot.yaml

```diff
@@ -12,3 +12,8 @@ capabilities:
       - banner
       - video
       - native
+userSync:
+  redirect:
+    url: "https://sync.adotmob.com/cookie/pbs?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r={{.RedirectURL}}"
+    userMacro: "{amob_user_id}"
+    redirectUrl: "{{.ExternalURL}}/setuid?bidder={{.SyncerKey}}&f={{.SyncType}}&uid={{.UserMacro}}"
```

Comment:
You should not define the `redirectUrl`. That's reserved for hosts to configure since it's a redirect back to the host's server.